src/components/Select/Select.tsx

```diff
@@ -80,28 +116,41 @@ const Select = React.forwardRef<HTMLSelectElement, SelectProps>(
     }
 
     return (
-      <select
-        name={name}
-        id={id}
-        className={bem("select", modifiers, blockName, [className])}
-        aria-required={isRequired}
-        disabled={disabled}
-        aria-label={ariaLabel}
-        aria-labelledby={ariaLabelledBy}
-        value={selectedOption}
-        ref={ref}
-        onBlur={onBlur}
-        onChange={onChange}
-        {...attributes}
-      >
-        {React.Children.map(children, (child, key) => {
-          return React.cloneElement(child as React.ReactElement<any>, {
-            "aria-selected": children[key]
-              ? children[key].props.children === selectedOption
-              : false,
-          });
-        })}
-      </select>
+      <div className="select">
+        {labelText && showLabel && (
+          <Label htmlFor={id} optReqFlag={optReqFlag} id={id + `-label`}>
+            {labelText}
+          </Label>
+        )}
+        <select
+          name={name}
+          id={id}
+          className={bem("selectfield", modifiers, blockName, [className])}
+          aria-required={isRequired || required}
+          required={isRequired || required}
```

Comment:
Nitpick: `isRequired || required` appears three times. Consider making a variable for this.

Reply:
Done.